keyboards/basekeys/slice/rev0_rgb/lib/host_led_state_reader.c

```diff
@@ -0,0 +1,15 @@
+#include <stdio.h>
+#include "crkbd.h"
```

Comment:
I'm pretty sure this isn't a corne.....  :D
Also, I don't see these used in any keymap.  

All of these should be removed (except maybe the glcdfont.c file.